docs/DavidTereshchenko/calculator.html

```diff
@@ -0,0 +1,120 @@
+                                </li>
+                                <li class="menu__item">
+                                    <a href="index.html" class="menu__link">About</a>
+                                </li>
+                                <li class="menu__item">
+                                    <a href="login.html" class="menu__link">Login</a>
+                                </li>
+                            </ul>
+                        </nav>
+                    </div>
+                    <div class="header__burger">
+                        <span></span>
+                    </div>
+                </div>
+            </header>
+            <main class="page">
+                <div class="main__container">
+                    <div class="title__calculator">
+                        <h1>Calculate Games</h1>
+                    </div>
+                    <div class="calculator__container">
+                        <div class="number__container">
+                            <div class="number-1">
+                                <label for="firstNumber">First number</label>
+                                <input id="firstNumber" data-number-one="one" class="number__block one" name="number1" required>
+                            </div>
+                            <div class="operator__block">
+                                <label>
+                                    <select class="operator" data-operation="operator">
+                                        <option value="+">+</option>
+                                        <option value="-">-</option>
+                                        <option value="×">×</option>
+                                        <option value="/">/</option>
+                                        <option value="%">%</option>
+                                        <option value="**">**</option>
+                                    </select>
+                                </label>
+                            </div>
+                            <div class="number-2">
+                                <label for="secondNumber">Second number</label>
+                                <input id="secondNumber" data-number-two="two" class="number__block two" name="number2" required>
+                            </div>
+                            <div class="return">
+                                <button type="button" class="button" data-button="button-result">=</button>
+                            </div>
+                            <div class="result__block">
+                                <p class="resul__number result"></p>
+                                <span class="count-games" data-count-game></span>
+                            </div>
+                        </div>
+                        <div class="count-block" data-count-block="count-block">
+                            <div class="visual-counting">
```

Comment:
The div above is an unnecessary wrap. Please remove it and add `class="count-block" data-count-block="count-block"` to `visual-counting` block